json-path/src/main/java/com/jayway/jsonpath/Predicate.java

```diff
@@ -23,6 +24,8 @@
 
     boolean apply(PredicateContext ctx);
 
+    boolean check(TokenStack stack, int idx);
```

Comment:
What do you mean by functional interface?
Hunter
 


     On Monday, July 27, 2015 12:49 AM, kallestenflo <notifications@github.com> wrote:
   

 In json-path/src/main/java/com/jayway/jsonpath/Predicate.java:> @@ -23,6 +24,8 @@
>  
>      boolean apply(PredicateContext ctx);
>  
> +    boolean check(TokenStack stack, int idx);
I want Predicate to be a functional interface. This will not work well with java 8.—
Reply to this email directly or view it on GitHub.